Tests/Util/FilenameUtilsTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Vich\UploaderBundle\Tests\Util;
+
+use PHPUnit\Framework\TestCase;
+use Vich\UploaderBundle\Util\FilenameUtils;
+
+class FilenameUtilsTest extends TestCase
+{
+    /**
+     * @dataProvider spitNameByExtensionProvider
+     */
+    public function spitNameByExtension($filename, $basename, $extension)
+    {
+        $this->assertSame([$basename, $extension], FilenameUtils::spitNameByExtension($filename));
+    }
+
+    public function spitNameByExtensionProvider()
+    {
+        return [
+            ['Текстовый файл.txt', 'Текстовый файл', 'txt'],
```

Comment:
We have a lot of data providers and no one has something similar that you talking about

- https://github.com/dustin10/VichUploaderBundle/blob/dcb416194e54e75def9d96b65c232e4346499a42/Tests/Naming/PropertyNamerTest.php#L24
- https://github.com/dustin10/VichUploaderBundle/blob/dcb416194e54e75def9d96b65c232e4346499a42/Tests/Naming/OrignameNamerTest.php#L17

each set contains file name, basename and extension